src/main/java/seedu/address/logic/parser/ExpiryDateTrackerParser.java

```diff
@@ -70,6 +71,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case HelpCommand.COMMAND_WORD:
             return new HelpCommand();
 
+        case SortCommand.COMMAND_WORD:
+            return new SortCommandParser().parse(arguments);
```

Comment:
I realise now that the pattern above for this parser uses the space character. I think need change since we using '|' character